src/components/results/securityanalysis/security-analysis-result-tab.tsx

```diff
@@ -127,7 +130,10 @@ export const SecurityAnalysisResultTab: FunctionComponent<
             if (sortConfig) {
                 const { sortWay, colKey } = sortConfig;
                 queryParams['sort'] = {
-                    colKey: FROM_COLUMN_TO_FIELD[colKey],
+                    colKey:
+                        tabIndex === 0
+                            ? FROM_COLUMN_TO_FIELD.subjectId
+                            : FROM_COLUMN_TO_FIELD[colKey],
```

Comment:
bug here, if you try to sort on any column, it will always sort on subjectId for N results

Reply:
Fixed